index.html

```diff
@@ -596,6 +596,43 @@ <h2>On behalf of organisations</h2>
 
 			</article>
 
+			<article class='container' lang='pl'>
+
+				<p>Jesteśmy społecznością osób, którym nie jest obojętny rozwój oraz stan światowej sieci internetowej oraz wyrażąmy <strong>głęboki niepokój</strong> w związku z technologią <a href='https://ampproject.org/'>Accelerated Mobile Pages</a> („AMP” – przyspieszone mobilne strony internetowe). AMP to projekt, którzy rzekomo ma na celu poprawę wygody użytkowników podczas korzystania z sieci internetowej.</p>
+				
+				<p>Tymczasem AMP zatrzymuje użytkowników wewnątrz domeny Google oraz odciąga ruch z innych stron internetowych na korzyść Google. W skali miliardów użytkowników jest to działanie, które <strong>jeszcze bardziej umocni monopolistyczną pozycję Google w sieci internetowej</strong>.</p>
+				
+				<p>Zdajemy sobie sprawę z problemu powolnego ładowania się witryn internetowych w stosunku do własnościowych rozwiązań takich jak <a href='https://instantarticles.fb.com/'>Facebook Instant Articles</a> i <a href='https://developer.apple.com/news-publisher/'>Apple News</a>. Wydawcy treści (zwłaszcza w sektorze mediów informacyjnych) od dawna zmagają się z trudnymi wyborami i niesłusznymi pobudkami, to prowadzi do złych decyzji i kompromisów, a te z kolei skutkują pogorszeniem wygody użytkowników.</p>
+				
+				<p>Silniki wyszukiwania to potężne narzędzia, dzięki którym można rozwiązać ten problem. Jednakże Google zdecydowało stworzyć specjalną pozycję na samej górze wyników wyszukiwania (dla artykułów) oraz dodaje specjalną ikonę <em>błyskawicy</em> (dla pozostałych treści), ale tylko dla wydawców korzystających z technologii kontrolowanej przez Google, udostępnianej przez Google w ramach ich infrastruktury, w ramach adresu URL Google oraz wewnątrz interfejsu stworzonego przez Google. (<a href='https://developers.google.com/search/docs/guides/about-amp'>źródło</a>)</p>
```

Comment:
The first sentence omit very important fragment – "wield influence". I'd suggest something like:

Reply:
> Rules of quotation demands that the dot is after the braces

That's how mind-numbing translating ends. Thanks! : >